OpenRA.Game/Game.cs

```diff
@@ -872,7 +872,7 @@ public static int CreateLocalServer(string map)
 
 			server = new Server.Server(new IPEndPoint(IPAddress.Loopback, 0), settings, ModData, false);
 
-			return server.Port;
+			return new ConnectionAddress(IPAddress.Loopback.ToString(), server.Address.Port);
```

Comment:
This will obviously fail if the IPv4 loopback address isn't reachable. Using the hostname `localhost` should work for all cases.